plugins/ltac2/tac2ffi.mli

```diff
@@ -51,6 +51,9 @@ val arity_suc : 'a arity -> (valexpr -> 'a) arity
 
 val mk_closure : 'v arity -> 'v -> closure
 
+val annotate_closure : Tac2expr.frame -> closure -> closure
```

Comment:
Latest push generalized this from being specialized to `FrPrim` to handling any `frame`
also fixed the test suite output tests